src/main/js/controls/PairedResult/PairedResult.js

```diff
@@ -188,10 +191,101 @@ class PairedResult extends GraphContent {
      * @inheritDoc
      */
     resize(graph) {
+        const regionItm = [];
+        let localRegionSameFlag = false;
+
         if (utils.notEmpty(this.dataTarget.regions)) {
-            // If graph has more than 1 content then we dont load the regions
+            const regionList = this.dataTarget.regions;
+            const values = this.dataTarget.values;
+
+            //check if all region are there with respect to value
+            if (graph.config.isPairedDataProper === true) {
+                for (let i = 0; i < values.length; i++) {
+                    // eslint-disable-next-line max-depth
+                    for (const key in values[i]) {
```

Comment:
Do we need to loop through keys? I think low, mid and high are the only possible keys.

Reply:
we have to iterate and check if the region for each (high, mid and low ) is there or not bcz values can be multiple, and i don't want to hard code keys to check.

Reply:
Looking elsewhere how carbon handles such scenarios, it looks like they use the keys. @abhijit945 - what do you think?

Reply:
Essentially you dont need to do this.
Consider this, as a graph the only thing we need to consider is what regions are drawn. There is no difference between how a line region and a paired graph region are rendered, they are just `rect` with different `y` pos. 

For instance: 

* For 2 regions belonging to separate contents (_Line 1_ has `R1` and _Line 2_ has `R2`)
    * Compare `R1` element's `datum()` and `R2` element's `datum()`
* For 3 regions belonging to separate contents (_Line 1_ has `R1` and _Paired Result 1_ has `R2` and `R3`)
    * After R2 and R3 is rendered, we just sequentially compare the first element `R1` with all the rest (`R2 and R3` in our case), since there is only one scenario where we show the regions `i.e. R1, R2 and R3 should all have same regions in same axis`.

Reply:
we discussed  in the previous PR all the scenarios if high, mid and low are provided in the value,
But in the region we done have the region for say low in the case it will be missing in the dom too and if me make the decision on the dom bases that was giving us an improper decision (as the in the previous logic in previous PR) since the region for low is missing even though all other regions are same.

If all the region are same one region is missing the picking from dom will make the region show up which is not what we need same for the line too if we don't even provide the region for say line and for paired we have given all region same then in that case too DOM will give us improper information.

Reply:
I am not sure I understand this correctly. Are you concerned with some contents not having regions?

Reply:
yes i am concerned with some contents not having regions.

Reply:
This is already handled isnt it? The 3rd content in [this](https://engineering.cerner.com/carbon-graphs/#/regions/line/multiple-line) example doesnt have a region so everything is hidden.

Reply:
No this is not handled. 
It's like if the content is > 1  call hideallregion which is not proper .

Reply:
for that example even if the region is same for 2 content also it will not show the region.  
No case was handled earlier.